spec/workflow/intrinsic_function_spec.rb

```diff
@@ -589,30 +589,86 @@
     end
 
     describe "States.JsonToString" do
-      it "fetches values from the input" do
-        # this is not in the spec but automatic as part of the parser
```

Comment:
Removed this comment because I checked all of these in the simulator and they all work with literals.